src/web/src/components/LanguageSelector.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { createContext } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { createStyles, FormControl, Grid, InputLabel, MenuItem, Select } from '@material-ui/core';
+
+import changeLang from '../translations/i18n';
+
+export interface TranslationInterface {
+  currentLanguage: string;
+}
+
+export const translationInterface = createContext<TranslationInterface>({
+  currentLanguage: 'en',
+});
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    root: {
+      // backgroundColor: theme.palette.primary.main,
+      // margin: '1rem 2rem 1rem 2rem',
+      // width: '10rem',
+      // position: 'absolute',
+      // right: '0',
+    },
+  })
+);
+const LanguageSelector = () => {
+  const classes = useStyles();
+
+  // Default language to translate is en
+  const [language, setLanguage] = React.useState('');
+
+  const changeLanguage = (e: any) => {
```

Comment:
```suggestion
  const changeLanguage = (e: React.ChangeEvent<HTMLSelectElement>) => {
```

Reply:
^ Or you can `import { ChangeEvent } from 'react'` so  the `React.` part won't be needed here.